natlas-server/app/static/natlas.scss

```diff
@@ -34,6 +43,151 @@ $white: #fff;
     color: $natlas-soft-blue;
 }
 
+@media (prefers-reduced-motion: reduce) {
+    #inner {
+        animation: none;
+    }
+}
+
+@keyframes spin { 100% { -webkit-transform: rotate(360deg); transform: rotate(360deg); } }
+
+@media (prefers-reduced-motion: no-preference) {
+    #inner {
+        animation: spin 30s linear infinite;
+        transform-origin: center;
+        transform-box: fill-box;
+    }
+}
+
+@media (prefers-color-scheme: light) {
+    body {
+        background-color: $off-white;
+        color: $off-black;
+    }
+
+    a {
+        color: $off-black;
+    }
+
+    .landing-input,
+    .landing-input:focus {
+        background-color: $off-white-darker;
+        color: $off-black;
+    }
+
+    .host-row:nth-child(odd) {
+        background-color: $light-content-bg;
+        border-top-color: $off-white;
+    }
+
+    .host-row:nth-child(even) {
+        background-color: $light-content-bg-alt;
+        border-top-color: $off-white;
+    }
+
+    .showtoken {
+        color: $off-black;
+    }
+
+    .showtoken:hover {
+        color: $off-black-lightest;
+    }
+
+    .table {
+        color: $off-black;
+    }
+}
+
+@media (prefers-color-scheme: dark) {
+    body {
+        background-color: $off-black;
+        color: $off-white;
+    }
+
+    a {
+        color: $off-white;
+    }
+
+    .landing-input,
+    .landing-input:focus {
```

Comment:
Can we move this into page specific files? Webpack adds an import mechanism to load other files.